dask/array/core.py

```diff
@@ -2273,6 +2273,10 @@ def concatenate(seq, axis=0, allow_unknown_chunksizes=False):
     stack
     """
     n = len(seq)
+
+    if n == 1:
+        return seq[0]
+
```

Comment:
I think in this case it does. That check is to error if there are unknown chunk sizes which would fail if things weren't already aligned. Since there is nothing to actually align here (only a single array) I think we shouldn't fail.